controller/node_controller.go

```diff
@@ -357,27 +357,45 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 		}
 		// get disk available size
 		diskInfo, err := nc.getDiskInfoHandler(disk.Path)
+		readyCondition := types.GetDiskConditionFromStatus(diskStatus, types.DiskConditionTypeReady)
 		// if the file system has changed
```

Comment:
In fact, this comment should be put below. This clause is not `file system has changed`.